microshift_running_apps/microshift-operators.adoc

```diff
@@ -0,0 +1,13 @@
+:_content-type: ASSEMBLY
+[id="operators-with-microshift"]
+= Using Operators
+include::_attributes/attributes-microshift.adoc[]
+:context: operators-microshift
+
+toc::[]
+
+Operators integrate with Kubernetes APIs and CLI tools such as `kubectl` and `oc` commands. They provide the means of monitoring applications, performing health checks, managing over-the-air (OTA) updates, and ensuring that applications remain in your specified state.
```

Comment:
These examples are all relevant for OCP, but not so much for MicroShift. Maybe we can come up with some other examples. I expect the sorts of operators people will deploy to MicroShift will be more focused on managing applications or resources used by applications (deploying a database or message bus, for example).

Reply:
We can just start with your examples and iterate?